src/styles/blocks/card.scss

```diff
@@ -0,0 +1,65 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
```

Comment:
Move to the * selector and remove from all other places

Reply:
if i move this to the * selector, my stars and price are become like column